components/SnakeGame/Child/Child.js

```diff
@@ -0,0 +1,38 @@
+import Image from "next/image";
+import styled from "styled-components";
+import capybara from "../capybara.png";
+
+const StyledChild = styled.div`
+  position: absolute;
+  top: ${({ $topcoor }) => $topcoor};
+  left: ${({ $leftcoor }) => $leftcoor};
+  font-size: 1rem;
+  border: black;
+`;
+
+const StyledIconContainer = styled.div`
+  display: grid;
+  place-content: center;
+  width: 30px;
+  height: 30px;
+`;
+
+const StyledIcon = styled.span`
+  font-size: 15px;
+  rotate: ${(props) => (props.$gameOn ? "unset" : "calc(180deg)")};
+`;
+
+export default function Child({ childPosition, gameOn, pet }) {
```

Comment:
I was under the impression that Child describes it quite well because building a family is what the whole game is about. Maybe I could change it to PetChild?
PetIcon would be subpar because in a coming story the icons will be interchanged with images.

Reply:
Ah, I got your point but in React, "Child" generally refers to a child component, meaning a component nested within a parent component, which might cause some confusion.

A possible alternative could be **"FamilyMember"** or **"Companion"** to emphasise both the family and playful aspects.

 **"PetChild"** could also work well if the focus is on pets and children.👍